lib/rpc/uniJsonRpcProvider.ts

```diff
@@ -0,0 +1,496 @@
+import SingleJsonRpcProvider from './singleJsonRpcProvider'
+import { StaticJsonRpcProvider, TransactionRequest } from '@ethersproject/providers'
+import Debug from 'debug'
+import { isEmpty } from 'lodash'
+import { ChainId } from '@uniswap/sdk-core'
+import {
+  BlockTag,
+  BlockWithTransactions,
+  Filter,
+  Log,
+  TransactionReceipt,
+  TransactionResponse,
+} from '@ethersproject/abstract-provider'
+import { LRUCache } from 'lru-cache'
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber'
+import { Network } from '@ethersproject/networks'
+import { Deferrable } from '@ethersproject/properties'
+
+const debug = Debug('UniJsonRpcProvider')
```

Comment:
We have log.debug using bunyan library.

Then when you want to enable debug on your local, you can modify the log level to debug at https://github.com/Uniswap/routing-api/blob/main/lib/handlers/quote/injector.ts#L38.